src/ui/screens/onboarding/OnboardingStartSlideMobile.qml

```diff
@@ -13,6 +13,8 @@ import "qrc:/nebula/utils/MZUiUtils.js" as MZUiUtils
 ColumnLayout {
     id: root
 
+    property string telemetryScreenId: "network_permissions"
```

Comment:
How come this was not caught in the tests?

Reply:
Ack, is this the mobile stuff?

Reply:
Yep, mobile only 😅

Reply:
Classic.